WriteThrough-/ListTriggerSync.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Cosmos;
+using System.Collections;
+using System.ComponentModel;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Redis;
+using System.Collections.Generic;
+using StackExchange.Redis;
+using System.Linq;
+using Microsoft.Azure.Cosmos.Linq;
+using Container = Microsoft.Azure.Cosmos.Container;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Redis.Samples
+{
+    public record ListData
+    (
+        string id,
+        List<string> value
+    );
+    public static class WriteThrough
+    {
+        //redis connection string
+        public const string localhostSetting = "redisLocalhost";
+        private static readonly IDatabase cache = ConnectionMultiplexer.Connect(Environment.GetEnvironmentVariable(localhostSetting)).GetDatabase();
+
+        //connecting to CosmosDB
+        public const string Endpoint = "Endpoint";
+
+        [FunctionName(nameof(ListTrigger))]
+        public static void ListTrigger(
+            [RedisListTrigger(localhostSetting, "listTest")] string listEntry, [CosmosDB(
+                            databaseName: "databaseName",
+                            containerName: "containerName",
+                            Connection = "Endpoint" )]CosmosClient input,
+            ILogger logger)
+        {
+            Container db = input.GetDatabase("databaseName").GetContainer("containerName");
```

Comment:
I don't think this db changes, should we define it outside the function so it is only defined once? The class would not be static anymore.

Reply:
It uses the Endpoint from the CosmosClient input to make the Container db. Do you mean just declaring Contianer db by itself on the outside?

Reply:
Following up on this - Because the function has a CosmosOutput Binding, we have to define and declare the container inside of the function.

Reply:
Oh I see, sorry I issed the CosmosClient was used as well.